src/odemis/gui/comp/viewport.py

```diff
@@ -1520,7 +1520,7 @@ def _on_new_data(self, data):
         data (1D DataArray)
         """
         if data is not None and data.size:
-            wll, unit_x = spectrum.get_spectrum_range(data)
+            wll, unit_x = spectrum.get_spectrum_range(data) # unit_x in 'm' or 'px'
```

Comment:
description of variable unit_x is given, but maybe it is nice to also comment on wll?